resume-buddy/src/main/webapp/scripts/script.js

```diff
@@ -86,3 +86,7 @@ function openForm(page) {
     }
   });
 }
+
+function successMessage() {
+  alert("Thank you, your form has been submitted.");
```

Comment:
Nit: JS strings should be contained in single-quotes, ie `'Thank you, your form has been submitted.'`